lib/request-fix.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const {Ci} = require('chrome')
+const events = require('sdk/system/events')
+const gw = require('./gateways.js')
+
+/*
+const catman = Cc['@mozilla.org/categorymanager;1'].getService(CI.nsICategoryManager)
+// category ("net-channel-event-sinks")
+catman.addCategoryEntry(in string aCategory, in string aEntry, in string aValue, in boolean aPersist, in boolean aReplace)
+*/
+
+function fixupRequest (e) {
```

Comment:
While implementing #72 I did some cleanup in https://github.com/lidel/ipfs-firefox-addon/commit/c75a05d82c3896d9eba8c1627895bef800978697.

I did not notice any regressions, but would appreciate if you took a second look, perhaps I missed something.